themes/TinkerSoc/layouts/partials/footer.html

```diff
@@ -5,7 +5,7 @@
         <iframe id="fb_like_button" style="border:none;overflow:hidden;vertical-align:top;" src="https://www.facebook.com/plugins/like.php?href=https%3A%2F%2Fwww.facebook.com%2FTinkerSoc%2F&width=146&layout=button_count&action=like&size=large&show_faces=false&share=true&height=46&appId" width="146" height="46" scrolling="no" frameborder="0" allowTransparency="true"></iframe>
         <a href="https://twitter.com/TinkerSoc" class="twitter-follow-button" data-size="large" data-dnt="true" data-show-count="true" height="46">Follow @TinkerSoc</a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
       </div>
-      <p class="copy">Copyright &copy; TinkerSoc {{ .Now.Format "2006" }}</p>
+      <p class="copy">Copyright &copy; TinkerSoc {{ .Date.Format "2006" }}</p>

```

Comment:
Should this be in the request?

Reply:
I can remove it, but will need to make another PR just for this, as the site won't build otherwise - .Now has been removed.

Reply:
Is this a change in hugo?

Reply:
Yes as far as I am aware .Now has been deprecated - but I'll change it to lowercase .now instead in a seperate PR, and remove this change.